doc/source/examples/dem/sandpile-formation/sandpile-formation.rst

```diff
@@ -0,0 +1,310 @@
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set g                        = 0.0, -9.81
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = custom
+        set custom diameters                  = 0.0069 , 0.0089
+        set custom volume fractions           = 0.7   , 0.3
+        set number of particles               = 3000
+        set density particles                 = 1056
+        set young modulus particles           = 4000000
+        set poisson ratio particles           = 0.49
+        set restitution coefficient particles = 0.7
+        set friction coefficient particles    = 0.8
+        set rolling friction particles        = 0.3
+      end
+      set young modulus wall           = 4000000
+      set poisson ratio wall           = 0.49
+      set restitution coefficient wall = 0.7
+      set friction coefficient wall    = 0.8
+      set rolling friction wall        = 0.3
+    end
+
+.. note::
+
+  Only the value of the restitution coefficient was not given in the paper.
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection model parameters
+      subsection contact detection
+        set contact detection method                = dynamic
+        set dynamic contact search size coefficient = 0.8
+        set neighborhood threshold                  = 1.3
+      end
+      set particle particle contact force method = hertz_mindlin_limit_overlap
+      set particle wall contact force method     = nonlinear
+      set integration method                     = velocity_verlet
+      set rolling resistance torque method       = epsd_resistance
+      set f coefficient                          = 0.0 
+    end
+
+.. note::
+
+  Only when the Elastic-Plastic Spring-Dashpot (epsd) rolling resistance model is selected should f be specified (f=0 here).
+  More information regarding the DEM Model parameters is given in the Lethe documentation, i.e. `DEM Model Parameters <../../../parameters/dem/model_parameters.html>`_.
+
+Particle Insertion
+~~~~~~~~~~~~~~~~~~
+
+Particles are inserted in an insertion box in the upper part of the hopper. In this simulation, the hopper is filled through 12 insertion steps.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method                               = volume
+      set inserted number of particles at each time step = 250
+      set insertion frequency                            = 10000
+      set insertion box points coordinates               = -0.34, 0.7 : 0.34, 1.2
+      set insertion distance threshold                   = 1.5
+      set insertion maximum offset                       = 0.1
+      set insertion prn seed                             = 20
+    end
+
+.. note::
+
+  Due partly to the bi-dispersed simulation, changing the prn seed to a different value can lead to different results.
```

Comment:
```suggestion
  Due partly to the bi-dispersed partic,le size distribution, changing the prn seed to a different value can lead to different results since it changes the initial configuration of the particles.
```